# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/client-hints/accept-ch-feature-policy-navigation.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/client-hints/accept-ch-feature-policy-navigation.https.html.ini
--- a/testing/web-platform/meta/client-hints/accept-ch-feature-policy-navigation.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/client-hints/accept-ch-feature-policy-navigation.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,10 +1,10 @@
 [accept-ch-feature-policy-navigation.https.html]
   max-asserts: 2
   expected:
-    if (os == "android") and debug and webrender: ["OK", "TIMEOUT", "CRASH"]
+    if (os == "android") and not debug and webrender: ["OK", "TIMEOUT"]
+    if (os == "android") and not debug and not webrender: ["OK", "TIMEOUT", "CRASH"]
     if (os == "android") and debug and not webrender: ["OK", "TIMEOUT", "CRASH"]
-    if (os == "android") and not debug and not webrender: ["OK", "TIMEOUT", "CRASH"]
-    if (os == "android") and not debug and webrender: ["OK", "TIMEOUT"]
+    if (os == "android") and debug and webrender: ["OK", "TIMEOUT", "CRASH"]
   [Client hints loaded on cross-origin iframe request with feature policy.]
     expected: FAIL
 